docs/docs/examples/storage/aws/s3/CONFLUENCE.md

```diff
@@ -0,0 +1,68 @@
+# AWS S3 Attachments Storage
+
+Since 8.1.0 Confluence supports storing attachments in AWS S3. To enable this feature, define bucket name and AWS region in `confluence.s3AttachmentsStorage`, for example:
```

Comment:
```suggestion
Since 8.1.0 Confluence supports storing attachments in AWS S3. To enable this feature, update the image `tag` to `8.1.0` and define bucket name and AWS region in `confluence.s3AttachmentsStorage`, for example:
```